rules/RESPONSE-954-DATA-LEAKAGES-IIS.conf

```
@@ -30,7 +30,7 @@ SecRule TX:DETECTION_PARANOIA_LEVEL "@lt 1" "id:954012,phase:4,pass,nolog,tag:'O
 #
 
 # IIS default location
-SecRule RESPONSE_BODY "@rx [a-z]:\x5cinetpub\b" \
+SecRule RESPONSE_BODY "@rx [a-z]:[\x5c/]inetpub\b" \
```

- make the regex case-insensitive and remove the `lowercase` transformation

```suggestion
SecRule RESPONSE_BODY "@rx (?i)[a-z]:[\x5c/]inetpub\b" \
```